CHANGELOG.md

```diff
@@ -7,52 +7,54 @@ and this project adheres to [Semantic Versioning](http://semver.org/).
+This Release has significant breaking changes in all ready-to-use implementations. The inheritance flow was highly simplified, by merging all extra utilities, which a marketplace would typically expect, into a single abstract contract: `RMRKImplementationBase`. This contract replaces `RMRKCollectionMetadata`, `RMRKMintingUtils` contracts and `name` and `symbol` of all core implementations.
 
-The contract includes royalties, a method to update the royalties recipient, internal utils to keep tack of next token and asset Ids, and these getters which marketplaces typically expect:
+The contract includes royalties, a method to update the royalties recipient, internal utils to keep track of next token and asset IDs, and the following getters that marketplaces commonly expect:
 
 - `totalSupply`
 - `maxSupply`
 - `name`
 - `symbol`
 - `collectionMetadata`
 
-Implementations no longer share the same constructor data, they take only the minimum needed. This reduces the size of the contracts giving more room to custom logic on top.
+Implementations no longer share the same constructor data; they only accept the necessary data. This reduces the size of the contracts allowing for more custom logic within your smart contract.
 
-Core implementations also have breaking changes, as name and symbol were removed from them to keep them fully non-opinionated.
+Core implementations updates include breaking changes as well; names and symbols were removed from them to keep them fully non-opinionated.
 
 ### Added
 
-- Added Soulbound implementation versions for all ready to use implementations.
-- Added `CollectionUtils` with methods to get several data points in a single call. This is a utility contract that needs to be deployed only once per chain, as `RenderUtils`. `getPaginatedMintedIds` Was moved from render utils to this contract.
-- Added new Render Utils: `getTotalDescendants`, `hasMoreThanOneLevelOfNesting`
+- Added Soulbound implementation versions for all ready-to-use implementations.
+- Added `CollectionUtils` with methods to get several data points with a single call. This is a utility contract that only needs to be deployed once per chain, just like `RenderUtils`. `getPaginatedMintedIds` was moved from render utils to this contract.
+- Added new Render Utils methods: `getTotalDescendants`, `hasMoreThanOneLevelOfNesting`
 - Added `TokenPropertiesRepository`.
 - Added `TokenHolder` extension.
-- Added new networks on hardhat config: Sepolia, Polygon Mumbai, Polygon and Mainnet. Removed Goerli.
-- Started Changelog.
+- Added new networks to Hardhat config: Sepolia, Polygon Mumbai, Polygon, and Mainnet. Removed Görli.
+- Started keeping a Changelog.
 
 ### Changed
 
-- Recreated all ready to use implementations for premint and lazy minting with native token or custom ERC20.
+- Recreated all ready-to-use implementations for premint and lazy minting with a native token or custom ERC20 token.
```

Comment:
I don't like the "a" here. It would make sense to me on custom ERC20 token since there are multiple, but there's only one native token, this makes it look like there are various options.

Reply:
In that case, I'd use plural:

```suggestion
- Recreated all ready-to-use implementations for premint and lazy minting with native tokens or custom ERC20 tokens.
```